README.md

```diff
@@ -1,7 +1,7 @@
 Run:
 
 ```
-$ docker run --detach --net=host --cap-add=NET_ADMIN --cap-add=NET_RAW --volume /var/run/docker.sock:/var/run/docker.sock tozd/external-ip
+$ docker run --detach --net=host --cap-add=NET_ADMIN --cap-add=NET_RAW --volume /var/run/docker.sock:/var/run/docker.sock --volume /run/xtables.lock:/run/xtables.lock tozd/external-ip
```

Comment:
This line is getting long, can you break it into multiple lines (using `\`)?

Reply:
Done in the latest commit